core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1325,6 +1327,9 @@ boolean scheduleTask(Workflow workflow, List<Task> tasks) {
                     .filter(isSystemTask.negate())
                     .collect(Collectors.toList());
 
+            Monitors.recordTaskStarted(tasksToBeQueued.size());
+            Monitors.recordSystemTaskStarted(systemTasks.size());
```

Comment:
Not necessarily the best place to do this. The tasks can be re-queued under certain circumstances. Please move this to where the task is being marked as COMPLETED

Reply:
Moved the counters to `updateTask`  and `completeWorkflow` methods so that they are incremented on completed.